src/test/java/Problem2Test.java

```diff
@@ -1,14 +1,24 @@
 import org.junit.Test;
 
-import static org.junit.Assert.*;
-
 public class Problem2Test {
 
     // Example test
     // Okay to change
+
+
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
+
         // add your tests here
+        int[] data ={4, 34, 76, 435, 23, 14, 16};
+
+        testBubbleSort();
```

Comment:
-5 testing not done correctly.

Reply:
without proper test, you will loose most of the points for a problem in future homework. please make sure you know how this is done. find help from instructor or in the discord channel when in doubt.